### PR TITLE
Add missing "static" modifier for constants

### PR DESCRIPTION
**Pull request by jukzi**
To reduce memory per instance

### CODE REVIEW

File: ui/org.eclipse.pde.ui.templates/templates_3.1/helloOSGiService/java/DictionaryImpl.java [hunk 1]
```diff
@@ -5,7 +5,7 @@
 public class DictionaryImpl implements Dictionary {
 
 	private final Set<String> fWords = Set.of("$word1$", "$word2$", "$word3$");
```
Comment (HannesWell) [inline ui/org.eclipse.pde.ui.templates/templates_3.1/helloOSGiService/java/DictionaryImpl.java:7]: Why not make this static too?

File: ui/org.eclipse.pde.ui.templates/templates_3.1/helloOSGiServiceComponent/java/DictionaryImpl.java [hunk 2]
```diff
@@ -8,7 +8,7 @@
 public class DictionaryImpl implements Dictionary {
 
 	private final Set<String> fWords = Set.of("$word1$", "$word2$", "$word3$");
```
Comment (HannesWell) [inline ui/org.eclipse.pde.ui.templates/templates_3.1/helloOSGiServiceComponent/java/DictionaryImpl.java:10]: Static too?

**Review by HannesWell — COMMENTED** [review]
Looks good, thanks, but it looks like there is room for more.

File: ui/org.eclipse.pde.ui.templates/templates_3.1/helloOSGiServiceComponent/java/DictionaryImpl.java (hunk 2 above)
Comment (jukzi) [reply ui/org.eclipse.pde.ui.templates/templates_3.1/helloOSGiServiceComponent/java/DictionaryImpl.java:10]: done

**Comment by merks** [comment]
If they're static, then the names fWords and fLanguage don't follow the Java naming conventions for static final constants...

**Comment by HannesWell** [comment]
But this still needs qualifier bumps in 

- org.eclipse.pde.ds.ui
- org.eclipse.pde.ui.templates

**Comment by jukzi** [comment]
thanks @HannesWell  - aka "early bird" - for all the review an version bumps.

**Pull request merged**